packages/lesswrong/components/common/Header.jsx

```diff
@@ -210,6 +212,8 @@ class Header extends PureComponent {
             classes.headroom,
             { [classes.headroomPinnedOpen]: searchOpen }
           )}
+          onUnfix={() => this.setState({unFixed: true})}
```

Comment:
I'm confused about the relationship between terminologies here. Why is pinning different from fixing? They sound like the same thing.

Reply:
It's the terminology that headroom uses. 

Basically "fixed" means it's at a static position relative to your screen, "unfixed" means it has an absolute position, "pinned" means it's pinned to the top of your screen, and "unpinned" means its hidden. 

We never enter the "fixed" state, so the terminology is particularly confusing, but seems better to use the same terminology instead of creating conflicting ones.